app/(gcforms)/[locale]/(support)/support/actions.ts

```diff
@@ -0,0 +1,66 @@
+"use server";
+import { createTicket } from "@lib/integration/freshdesk";
+import { logMessage } from "@lib/logger";
+
+interface Submission {
+  name?: string;
+  email?: string;
+  request?: string;
+  description?: string;
+  language?: string;
+}
+
+export async function support({ name, email, request, description, language = "en" }: Submission) {
+  // No auth etc. checking since this is a public endpoint
+
+  //Mandatory fields
+  if (!name || !email || !request || !description) {
```

Comment:
server actions is intelligent enough that it will throw back an error to the calling function if a required field is not present.  If you modify the type to remove the undefined status from known required fields we won't need to do a "malformed request" check.

Reply:
Sounds good. Some time we should talk about "client" and "server" validation and how these patterns have probably changed.
My thinking was to do the main validation on the client and leave this check on the server just for form sake. A message could be returned but a thrown error with a generic error screen seemed good enough - since this should never happen